### PR TITLE
Button link

### PR DESCRIPTION
**Pull request by NatalyKokidko**

### CODE REVIEW

File: client/components/errorLabel/errorLabel.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<link rel="stylesheet" href="./errorLabel.css">
+<error-label id="label_error" class="error-message">Error</error-label>
```
[the remark this answers is not in the document]
Comment (tbazko) [reply client/components/errorLabel/errorLabel.html:2]: also is not related to our task

File: client/components/errorLabel/errorLabel.css [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-    background: #FFF2F6;
+    background: var(--invalid-background);
     border-radius: 8px;
-    font-family: 'IBM Plex Sans';
-    font-style: normal;
-    font-weight: 600;
-    font-size: 14px;
-    color: #B30038;
     border: none;
     }
```
Comment (tbazko) [inline client/components/errorLabel/errorLabel.css:14]: this changes are also not related to button link can you remove this from PR?

Comment (NatalyKokidko) [reply client/components/errorLabel/errorLabel.css:14]: Good afternoon, good.

**Comment by tbazko** [comment]
@NatalyKokidko I would merge this, but you still didn't remove not related files

**Review by anynikulina — APPROVED** [review]
